.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,17 @@
+name: Node.js CI
+on: [push, pull_request]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x]
```

Comment:
As we have no `engine` in the `package.json`, I think we can just run on `12.x`.

Reply:
I personally prefer always testing with different versions commonly in use to catch any unexpected incompatibility bugs. Not sure how relevant that is for this specific project. Your call.

Reply:
Let's keep them both for now 👍